### ISSUE TITLE
Implementar Hosted Services

### ISSUE THREAD

**Issue by alexsandro-xpt** [patrickreinan/programapontos#3]
Seria interessante implementar os projetos console app usando Hosted Services porque eles fazem backgrond task como é mencionado em https://docs.microsoft.com/en-us/aspnet/core/fundamentals/host/hosted-services

Segue o ponto de mereceria usar o recurso.
https://github.com/patrickreinan/programapontos/blob/acf7fab318ac646922c7fe317c60518777a18424/ProgramaPontos.EventHandler.Sinc/Program.cs#L14

**Comment by patrickreinan** [comment]
Concordo com você. Como este projeto era apenas para fins de demonstração do CQRS com o EventSourcing eu acabei deixando alguns detalhes de lado. É uma melhoria que farei.

**Comment by patrickreinan** [comment]
Excelente!!!
